### DISCUSSION TITLE
Importing Spoke Projects from Hubs into Hubs Clouds Breaks Content Metadata.

### DISCUSSION THREAD

**Discussion by NovySan** [mozilla/hubs#3876] [Spoke Questions] [answered]
Exporting a project from regular hubs and importing it into hubs cloud on our AWS stack breaks the metadata if there are meshes from Google Poly or Sketchfab.

I can export a combined mesh as a .glb, but then the constituent parts are no longer editable and the broken metadate links throw an error when trying to publish the hubs cloud project.

EDIT: Setting up Google Poly API credentials fixed this issue.

**Comment by Softvision-GeluHaiduc** [comment, marked as answer]
Happy to hear that you could find the solution to your issue.
Marking this as answered to move it out of our pipeline, unmark the answer if you need any more information. Thank you!
